### ISSUE TITLE
Investigate why some Spanish / French stopwords don't get removed

### ISSUE THREAD

**Issue by pypt** [berkmancenter/mediacloud#209]
Fernando writes:

> I got a note the other day from Connie (Global Voices) telling me she was finding stop words in some word clouds in French. I was just running a query in Spanish and also got a bunch of them: 
> `https://dashboard.mediacloud.org/#query/["Puigdemont AND tags_id_media:9361236\n"]/[{}]/["2016-10-17"]/["2017-10-16"]/[{"uid":1,"name":"Puigdemont AND tags_id...","color":"e14c11"}]`

**Comment by connieimdialog** [comment]
hi!  i am uploading a spreadsheet w
[mediacloud-wordcount Filter Out.xlsx](https://github.com/berkmancenter/mediacloud/files/1437221/mediacloud-wordcount.Filter.Out.xlsx)
 with what we consider potential candidates in French and Spanish, for further discussion

**Comment by pypt** [comment]
Added Connie's proposed French and Spanish stopwords. Made `::Solr::WordCounts` remove stopwords from both English, story and sentence languages (instead of just English and sentence languages), `de` is still there, investigating further.

**Comment by pypt** [comment]
Found the culprit: some (quite a few of) sentences for the `Puigdemont` query were being identified as being Catalan (either because they were actually Catalan or were being misidentified Spanish sentences).

As a temporary (and politically incorrect) fix, I've added Catalan language support module which uses a set of Spanish + Catalan stopwords. I'll add proper Catalan support later as a separate issue.

For now, there seems to be some weird cache in place for the query so for the old `Puigdemont` query you'll get the same old results with `de` (@hroberts, @rahulbot which cache could it be?), but a query with some extra spaces in front (`  Pugdemont`) seems to work fine:

```
https://dashboard.mediacloud.org/#query/["  puigdemont AND tags_id_media:9361236"]/[{}]/["2016-10-17"]/["2017-10-16"]/[{"uid":3,"name":"Puigdemont AND tags_id...","color":"e14c11"}]
```
